VTEX - Payment Provider Protocol.json

```diff
@@ -1855,160 +2224,40 @@
                     "Configuration Flow"
                 ],
                 "summary": "Create Authorization Token",
-                "description": "Create token that will be used to identify the same context  when we redirect the merchant to your application.",
+                "description": "Creates token that will be used to identify the same context when we redirect the merchant to your application. \r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
"Redirect the merchant"? What context is this request used in? This description should be more specific.